diff -ur Bluespec-2013.09.beta1/lib/board_support/bluenoc/xilinx/KC705/verilog/k7_pcie_v1_6/source/xilinx_k7_pcie_wrapper.v Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/KC705/verilog/k7_pcie_v1_6/source/xilinx_k7_pcie_wrapper.v
--- Bluespec-2013.09.beta1/lib/board_support/bluenoc/xilinx/KC705/verilog/k7_pcie_v1_6/source/xilinx_k7_pcie_wrapper.v	2013-09-03 00:50:40.000000000 -0400
+++ Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/KC705/verilog/k7_pcie_v1_6/source/xilinx_k7_pcie_wrapper.v	2013-10-29 15:49:14.000000000 -0400
@@ -7,7 +7,15 @@
 
 module xilinx_k7_pcie_wrapper #(
                                 parameter            PL_FAST_TRAIN = "FALSE",
-                                parameter            PCIE_EXT_CLK  = "TRUE"
+                                parameter            PCIE_EXT_CLK  = "TRUE",
+// xbsv
+				parameter [31:0]  BAR0 = 32'hFFFF8000,
+				parameter [31:0]  BAR1 = 32'h00000000,
+				parameter [31:0]  BAR2 = 32'h00000000,
+				parameter [31:0]  BAR3 = 32'h00000000,
+				parameter [31:0]  BAR4 = 32'h00000000,
+				parameter [31:0]  BAR5 = 32'h00000000
+// xbsv
                                 )
 (
  
@@ -275,7 +283,15 @@
    
    k7_pcie_v1_6 #(
                   .PL_FAST_TRAIN       ( PL_FAST_TRAIN ),
-                  .PCIE_EXT_CLK        ( PCIE_EXT_CLK )
+                  .PCIE_EXT_CLK        ( PCIE_EXT_CLK ),
+// xbsv
+		  .BAR0                ( BAR0 ),
+		  .BAR1                ( BAR1 ),
+		  .BAR2                ( BAR2 ),
+		  .BAR3                ( BAR3 ),
+		  .BAR4                ( BAR4 ),
+		  .BAR5                ( BAR5 )
+// xbsv
                   )
    k7_pcie_v1_6_i
      (
Only in Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/KC705/verilog/k7_pcie_v1_6/source: xilinx_k7_pcie_wrapper.v~
diff -ur Bluespec-2013.09.beta1/lib/board_support/bluenoc/xilinx/VC707/verilog/v7_pcie_v1_7/source/xilinx_v7_pcie_wrapper.v Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/VC707/verilog/v7_pcie_v1_7/source/xilinx_v7_pcie_wrapper.v
--- Bluespec-2013.09.beta1/lib/board_support/bluenoc/xilinx/VC707/verilog/v7_pcie_v1_7/source/xilinx_v7_pcie_wrapper.v	2013-09-03 00:50:40.000000000 -0400
+++ Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/VC707/verilog/v7_pcie_v1_7/source/xilinx_v7_pcie_wrapper.v	2013-10-29 15:49:43.000000000 -0400
@@ -8,6 +8,14 @@
 module xilinx_v7_pcie_wrapper #(
                                 parameter            PL_FAST_TRAIN = "FALSE",
                                 parameter            PCIE_EXT_CLK  = "TRUE"
+// xbsv
+				parameter [31:0]  BAR0 = 32'hFFFF8000,
+				parameter [31:0]  BAR1 = 32'h00000000,
+				parameter [31:0]  BAR2 = 32'h00000000,
+				parameter [31:0]  BAR3 = 32'h00000000,
+				parameter [31:0]  BAR4 = 32'h00000000,
+				parameter [31:0]  BAR5 = 32'h00000000
+// xbsv
                                 )
 (
  
@@ -276,6 +284,14 @@
    v7_pcie_v1_7 #(
                   .PL_FAST_TRAIN       ( PL_FAST_TRAIN ),
                   .PCIE_EXT_CLK        ( PCIE_EXT_CLK )
+// xbsv
+		  .BAR0                ( BAR0 ),
+		  .BAR1                ( BAR1 ),
+		  .BAR2                ( BAR2 ),
+		  .BAR3                ( BAR3 ),
+		  .BAR4                ( BAR4 ),
+		  .BAR5                ( BAR5 )
+// xbsv
                   )
    v7_pcie_v1_7_i
      (
Only in Bluespec-2013.09.beta1.xbsv1/lib/board_support/bluenoc/xilinx/VC707/verilog/v7_pcie_v1_7/source: xilinx_v7_pcie_wrapper.v~
