--- sources/imagecapture/ImageCaptureWrapper.bsv.001	2013-09-28 09:41:17.872661411 -0400
+++ sources/imagecapture/ImageCaptureWrapper.bsv	2013-09-28 09:46:53.916791401 -0400
@@ -34,7 +34,7 @@
 
 endinterface
 
-module mkImageCaptureWrapper(ImageCaptureWrapper);
+module mkImageCaptureWrapper#(Clock hdmi_clk)(ImageCaptureWrapper);
     Reg#(Bit#(TLog#(2))) axiSlaveWS <- mkReg(0);
     Reg#(Bit#(TLog#(2))) axiSlaveRS <- mkReg(0); 
     CoreIndicationWrapper coreIndicationWrapper <- mkCoreIndicationWrapper();
